front-end/src/components/Contents/Posts/Posts.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import './style.scss';
+
+export default function Posts() {
+
+  const cards = [
+    {
+      'title' : 'Card Title',
+      'text' : 'This is a longer card with supporting text below as a natural lead-in to additional content.'
+    }, {
+      'title' : 'Card Title',
+      'text' : 'This is a longer card with supporting text below as a natural lead-in to additional content.'
+    }, {
+      'title' : 'Card Title',
+      'text' : 'This is a longer card with supporting text below as a natural lead-in to additional content.'
+    }, {
+      'title' : 'Card Title',
+      'text' : 'This is a longer card with supporting text below as a natural lead-in to additional content.'
+    }
+  ];
+  const cnt = [1, 2, 3];
+
+  return (
+    <div id='posts'>
+      {cnt.map((c) => (
+        <div className='row'>
+          {cards.map((card) => (
+            <div className='card'>
+              <img src={`${process.env.PUBLIC_URL}/image/card-image.png`} alt='thumnail' />

```

Comment:
@Taiga17 -san
```JS
process.env.PUBLIC_URL
```
Please do not that, sorry  because I haven't yet introduce how to define and access .env
We need to follow the config as below and doc: https://create-react-app.dev/docs/adding-custom-environment-variables/

<img width="1211" alt="Screen Shot 2022-03-11 at 09 28 24" src="https://user-images.githubusercontent.com/86216577/157790576-63fb83f1-73a0-459b-bfde-52a32c7e27a5.png">

And define .env as below.

<img width="1331" alt="Screen Shot 2022-03-11 at 09 29 07" src="https://user-images.githubusercontent.com/86216577/157790660-64d38ab1-fe2d-4929-b9fd-124a82657d6f.png">